src/components/contribution/ContributionAmount.vue

```diff
@@ -93,7 +90,11 @@
           justify-center
           text-primary
           font-semibold
+          hover:bg-link-10
         "
+        :class="{
+          'bg-link text-white hover:bg-link-110': amount === modelValue,
```

Comment:
I think `hover:bg-link-110` probably just wants to be `hover:bg-link` to match the change in `ContributionPeriod`?

Alternatively so you don't have to override the hover state you could conditionally set `hover:bg-link-10`, e.g.
```
:class="amount === modelValue ? 'bg-link text-white' : 'hover:bg-link-10'
```